ludwig/benchmarking/resource_usage_tracker.py

```diff
@@ -32,37 +37,40 @@
 STOP_MESSAGE = "stop"
 
 
-def monitor(
-    queue: Queue, info: Dict[str, Any], output_dir: str, logging_interval: int, cuda_is_available: bool
-) -> None:
+def monitor(queue: Queue, info: Dict[str, Any], logging_interval: int, cuda_is_available: bool) -> None:
     """Monitors hardware resource use.
 
-    Populate `info` with system specific metrics (GPU, CPU, RAM) at a `logging_interval` interval and saves the output
-    in `output_dir`.
+    Populate `info` with system specific metrics (CPU/CUDA, CPU/CUDA memory) at a `logging_interval` interval
+    and saves the output in `output_dir`.
```

Comment:
`output_dir` seems to be removed from the function header, but is still referred to in the docstring here